js/apps/system/_admin/aardvark/APP/aardvark.js

```diff
@@ -294,6 +294,7 @@ authRouter.get('/query/download/:user', function (req, res) {
     res.throw('not found');
   }
 
+  // TODO: properly escape name here
   res.attachment(`queries-${db._name()}-${user.user}.json`);
```

Comment:
Would doing a simple `.replace(/[^a-z0-9]/gi, "_")` or similar be enough? It's not going to handle umlauts and non-Latin characters gracefully but it should prevent bad things from happening.

Reply:
Adjusted